src/index.js

```diff
@@ -0,0 +1,2 @@
+import './styles/style.css';
+import './modules/app.js';
```

Comment:
- [ ] As the last code reviewer said please fix the compiled error.

![image](https://user-images.githubusercontent.com/38635158/168419296-f2fbccc4-4e74-4d7f-b493-0e34e5e13ac7.png)